rascal-vscode-extension/src/auto-jvm/downloaders.ts

```diff
@@ -176,12 +176,23 @@ export async function identifyLatestTemurinLTSRelease(version: number, arch: Tem
         throw new Error("Adoptium returned no releases");
     }
     const rel =releases.versions[0];
+    if (!rel) {
+        throw new Error("Missing versions in api result" + rel);
```

Comment:
Cryptic error for user. Include talk about the jdk

Reply:
good point, I've updated it.